.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/*
+*.pyc
+*.conf
```

Comment:
We might want to use something like https://github.com/github/gitignore/blob/master/Python.gitignore to prepoulate this file.